com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/lightlistbuild-clustered.compute

```diff
@@ -10,6 +10,18 @@
 #pragma kernel TileLightListGen_DepthRT_SrcBigTile_Oblique          LIGHTLISTGEN=TileLightListGen_DepthRT_SrcBigTile_Oblique            ENABLE_DEPTH_TEXTURE_BACKPLANE          USE_TWO_PASS_TILED_LIGHTING				USE_OBLIQUE_MODE
 #pragma kernel TileLightListGen_DepthRT_MSAA_SrcBigTile_Oblique     LIGHTLISTGEN=TileLightListGen_DepthRT_MSAA_SrcBigTile_Oblique       ENABLE_DEPTH_TEXTURE_BACKPLANE			MSAA_ENABLED        USE_TWO_PASS_TILED_LIGHTING			USE_OBLIQUE_MODE
 
+#pragma kernel TileLightListGen_NoDepthRT_LE                LIGHTLISTGEN=TileLightListGen_NoDepthRT_LE      PLATFORM_LANE_COUNT_32
```

Comment:
We should simply use a multi compile here, like for the other one. I don't see the need for a "LE" thing here.

Reply:
I guess I have a global misunderstand about what "LE" mean. it is just to have 32 vs 64? Or does it mean Lightweight?

Reply:
LE was originally a short for something of LowEnd, for DeviceInfo to prefer a specific suffix if they happen to exist. I thought about renaming it as _32 as well.

ComputeGgxIblSampleData.compute has nothing to do with PLATFORM_LANE_COUNT_32, so it made sense to use _LE naming convention there to differentiate MAX_IBL_SAMPLE_CNT=89/MAX_IBL_SAMPLE_CNT=34 kernel pickup to runtime.